nbs/guides/Setting_up_social_cards.ipynb

```diff
@@ -27,9 +32,10 @@
     "https://opengraph.githubassets.com/some-random-stuff-to-refresh-or-timestamp/{user}/nbdev_mkdocs_tutorial\n",
     "```\n",
     "\n",
-    "#### Customization\n",
+    "## Generating new social cards\n",
+    "\n",
     "\n",
-    "To customize the social share image for your project, use the `nbdev_mkdocs social-image generate` CLI command. To view the current configuration options, run the following command in the terminal.\n",
+    "To customize and generate a new social share image for your project, you can use the `nbdev_mkdocs social-image generate` CLI command. To view the current configuration options, run the following command in the terminal.\n",
```

Comment:
For customizing and generating a new social share image for your project, you can use the `nbdev_mkdocs social-image generate` CLI command.

Reply:
Fixed